app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -0,0 +1,80 @@
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+.steps-container {
+  overflow: hidden;
+  margin: 0px;
+  padding: 0px;
+  white-space: nowrap;
+  border-left: 2px solid;
+  border-right: 2px solid;
+  width: 100%;
+  counter-reset: steps;
+}
+.steps {
+  position: relative;
+  display: inline-block;
+  left: -28px; /* -2px default + 26px offset to hide skewed area on the left side of first element*/
+  height: 50px;
+  line-height: 50px;
+  margin-left: 0px;
+  margin-right: 0px;
+  counter-increment: steps;
+  cursor: pointer;
+  transition: background 1s;
+  min-height: 30px;
+  width: 100px;
+}
+
+.steps:after,
+.steps:before {
+  position: absolute;
```

Comment:
Properties should be ordered background, border-bottom, border-left, border-right, border-top, content, height, left, position, width